### PR TITLE
Make read only state reflect in UI for primary user store

### PR DESCRIPTION
**Pull request by DimalChandrasiri**
## Purpose
Fixes https://github.com/wso2/product-is/issues/10762

### CODE REVIEW

File: apps/console/src/features/core/api/user-store.ts [hunk 1]
```diff
@@ -29,6 +29,35 @@ const httpClient = IdentityClient.getInstance()
     .httpRequest.bind(IdentityClient.getInstance())
     .bind(IdentityClient.getInstance());
 
+/**
+ * Gets details of the primary user store.
+ *
```
Comment (brionmario) [inline apps/console/src/features/core/api/user-store.ts:34]:
```suggestion
```

File: apps/console/src/features/core/api/user-store.ts [hunk 2]
```diff
@@ -29,6 +29,35 @@ const httpClient = IdentityClient.getInstance()
...
+ *
+ * @return {Promise<any>} response.
+ */
+export const getPrimaryUserStore = (): Promise<any> => {
```
Comment (brionmario) [inline apps/console/src/features/core/api/user-store.ts:38]: Is there a possibility to add the return type here?

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/core/utils/user-store-utils.ts [hunk 3]
```diff
@@ -83,13 +83,30 @@ export class SharedUserStoreUtils {
         });
     }
 
+    /**
+     * The following method will fetch the primary user store details.
+     */
+    public static async getPrimaryUserStore(): Promise<any> {
+        return getPrimaryUserStore().then((response) => {
```
Comment (NipuniBhagya) [inline apps/console/src/features/core/utils/user-store-utils.ts:90]: Shall we properly handle the error?

**Review by NipuniBhagya — APPROVED** [review]

File: apps/console/src/features/core/api/user-store.ts (hunk 2 above)
Comment (DimalChandrasiri) [reply apps/console/src/features/core/api/user-store.ts]: Fixed with commit 0a71bfe

File: apps/console/src/features/core/utils/user-store-utils.ts (hunk 3 above)
Comment (DimalChandrasiri) [reply apps/console/src/features/core/utils/user-store-utils.ts:95]: Fixed with 2c6c360

**Pull request merged**